b3sum/src/main.rs

```diff
@@ -537,7 +537,7 @@ fn check_one_line(line: &str, args: &Args) -> bool {
     }
 }
 
-fn check_one_checkfile(path: &Path, args: &Args, some_file_failed: &mut bool) -> Result<()> {
+fn check_one_checkfile(path: &Path, args: &Args, files_failed: &mut i64) -> Result<()> {
```

Comment:
Could we make this a `u64`? I'm thinking negative values wouldn't make sense here.